CHANGELOG.md

```diff
@@ -33,6 +33,12 @@
   ```
   
   In Dart SDK `>=1.25.0` this can be relaxed as `part of` can refer to a path.
+  To opt-in, `GeneratorBuilder` now has a new flag, `requireLibraryDirective`.
```

Comment:
I figured adding this flags gives users the chance to make that choice for themselves versus waiting for us to do the right thing ;)